src/utiltime.cpp

```diff
@@ -81,3 +82,18 @@ std::string DateTimeStrFormat(const char* pszFormat, int64_t nTime)
     ss << boost::posix_time::from_time_t(nTime);
     return ss.str();
 }
+
+std::string DurationToDHMS(int64_t nDurationTime)
+{
+    int seconds = nDurationTime % 60;
+    nDurationTime /= 60;
+    int minutes = nDurationTime % 60;
+    nDurationTime /= 60;
+    int hours = nDurationTime % 24;
+    int days = nDurationTime / 24;
+    if((hours == 0)&&(days == 0))
```

Comment:
agree, rewritten this part, see https://github.com/dashpay/dash/pull/776/commits/3764e6869ad67ff75e7928ebe06c28cba667e250#diff-8e9b29a6133dd7bf43643c37c5c4689aR94